src/cozmo/action.py

```diff
@@ -189,17 +189,17 @@ def _recv_msg_robot_completed_action(self, evt, *, msg):
             logger.warn('Received "running" action notification for action=%s', self)
             self._set_failed('running', 'Action was still running')
 
-        elif result == types.NOT_STARTED:
+        elif result == types.FAILURE_NOT_STARTED:
```

Comment:
I think the naming on the left (without FAILURE_) was to match the latest code in cozmo-one master, or did that change back again? (python-sdk master is still like the change on the right).